src/sorcha/utilities/baseLightCurve.py

```diff
@@ -0,0 +1,62 @@
+from abc import ABC, abstractmethod
+import numpy as np 
+
+timeColumn = 'FieldMJD'
+
+class baseLightCurve(ABC):
```

Comment:
Do we expect anyone to ever actually use the `BaseLightCurve` class for doing work? If not, perhaps it would make sense to rename this to something like `AbstractLightCurve`?

If that makes sense, you could also replace lines 29-31 with lines 56-61.